### PR TITLE
Zarmada/working/adts 143

### PR DESCRIPTION
**Pull request by JoseZarmada**
### Testing 🧪
 > [ Add any special instructions needed to test or view your changes such as environment URLs or other config details. ]

### Checklist ✔️
- [ ] Linked associated issue (if present)
- [ ] Added relevant labels
- [ ] Requested developer reviews
- [ ] Request UI review from design / PM
- [ ] Verify all code checks are passing

### CODE REVIEW

File: src/Components/OATGraphViewer/OATGraphViewer.tsx [hunk 1]
```diff
@@ -68,11 +69,13 @@ const OATGraphViewer = ({ state, dispatch }: OATGraphProps) => {
     const [elements, setElements] = useState(
         !storedElements ? [] : storedElements
     );
-    const defaultPosition = 100;
+    let defaultPositionX = 100;
+    let defaultPositionY = 100;
```
Comment (jamescarpinter) [inline src/Components/OATGraphViewer/OATGraphViewer.tsx:73]: These will get reset to the default values each time the component re-renders.

Maybe we return these from the positionLookUp method instead?

**Review by jamescarpinter — CHANGES_REQUESTED** [review]
Minor comment only. Thanks!

File: src/Components/OATGraphViewer/OATGraphViewer.tsx (hunk 1 above)
Comment (JoseZarmada) [reply src/Components/OATGraphViewer/OATGraphViewer.tsx:73]: I was counting on that to happen, so, the LookUp method will fill the gaps.

File: src/Components/OATGraphViewer/OATGraphViewer.tsx [hunk 2]
```diff
@@ -653,6 +655,44 @@ const OATGraphViewer = ({ state, dispatch }: OATGraphProps) => {
...
+    const positionLookUp = (node = null) => {
+        let defaultPositionX = 100;
+        let defaultPositionY = 100;
+        const areaDistanceX = 250;
+        const areaDistanceY = 80;
+        const maxWidth = 800;
+        const defaultPosition = 100;
+        const minWidth = 300;
+        const minHeight = 100;
+        const lookUpElements = node ? [...elements, ...node] : [...elements];
```
Comment (jamescarpinter) [inline src/Components/OATGraphViewer/OATGraphViewer.tsx:667]: Is `node` the best name for this param? It looks like it's an array of elements?

**Review by jamescarpinter — APPROVED** [review]
Please see minor comment re renaming a parameter. If it's fixed then this is good to go.

**Pull request merged**